manifests/j/Jidifudi/GT4T/8.49.250222/Jidifudi.GT4T.installer.yaml

```
@@ -0,0 +1,30 @@
+# Automatically updated by the winget bot at 2025/Feb/21
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: Jidifudi.GT4T
+PackageVersion: 8.49.250222
+MinimumOSVersion: 10.0.0.0
+InstallerType: inno
+Scope: machine
+UpgradeBehavior: install
+Installers:
+- Architecture: x86
+  InstallerUrl: https://gt4t.net/apps/gt4t_win_32_bit/GT4T_setup.exe
+  InstallerSha256: 2953247B768E21EB5A30ABC141D9DD1C0AD78FB76D426E4F593DFB3BF6F0E18B

```

```suggestion
 InstallerSha256: DC5CDB0B433BF45FD8FFF224704B56B6388D39AD804CA61DA5C64127B587D8EF
```

(Automated response - build 982.)